sdk/identity/Azure.Identity/src/InteractiveBrowserCredential.cs

```diff
@@ -78,7 +78,9 @@ internal InteractiveBrowserCredential(string tenantId, string clientId, TokenCre
 
             Pipeline = pipeline ?? CredentialPipeline.GetInstance(options);
 
-            Client = client ?? new MsalPublicClient(Pipeline, tenantId, clientId, "http://localhost", options as ITokenCacheOptions);
+            var redirectUrl = (options as InteractiveBrowserCredentialOptions)?.RedirectUri?.AbsoluteUri ?? Constants.DefaultRedirectUrl;
```

Comment:
Shouldn't we fail if URI isn't absolute?

Reply:
`AbsoluteUri` will throw if the uri isn't absolute. Do you feel this is insufficient? We could add a check on the set operator of `InteractiveBrowserCredential.RedirectUri`, but personally I think failing at construction time would be acceptable. Thoughts?

Reply:
> `AbsoluteUri` will throw if the uri isn't absolute.

Ah, ok, that's enough